ui/partials/main/intermittent.html

```diff
@@ -76,6 +76,11 @@
         <input type="text" ng-model="modalDependsOn" placeholder="Depends on" uib-tooltip="Comma-separated list of bugs" tooltip-placement="bottom" />
         <input type="text" ng-model="modalSeeAlso" placeholder="See also" uib-tooltip="Comma-separated list of bugs" tooltip-placement="bottom" />
       </div>
+
+      <div ng-hide="crashSignatures.length === 0" id="modalCrashSignatureDiv">
```

Comment:
I think you could just do:

    <div ng-hide="!crashSignatures.length"

or:

    <div ng-show="crashSignatures.length"